### PR TITLE
Remove Profiles from AU Core R1 Scope prior to March Ballot

### LINKED ISSUE #106: Remove Profiles from AU Core R1 Scope prior to March Ballot
As per [2024-02-23 AU Core TDG Agenda/Minutes](https://confluence.hl7.org/pages/viewpage.action?pageId=220704102). Remove the follow profiles:

1. [AU Core DiagnosticReport](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-diagnosticreport.html)
2. [AU Core DocumentReference](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-documentreference.html)
3. [AU Core HealthcareService](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-healthcareservice.html)
4. [AU Core BMI](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-bmi.html)
5. [AU Core Diagnostic Imaging Result Observation](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-diagnosticresult-imag.html)
6. [AU Core Observation](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-observation.html)
7. [AU Core Oxygen Saturation](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-oxygensat.html)
8. [AU Core Vitals Panel](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-vitalspanel.html)
9. [AU Core ServiceRequest](https://build.fhir.org/ig/hl7au/au-fhir-core/StructureDefinition-au-core-servicerequest.html)

Other changes:
- Remove Note on Consent from [Profiles and Extensions](https://build.fhir.org/ig/hl7au/au-fhir-core/profiles-and-extensions.html#profiles-and-extensions)
- Update [AU Core Client CapabilityStatement](https://build.fhir.org/ig/hl7au/au-fhir-core/CapabilityStatement-au-core-client.html)
- Update [AU Core Server CapabilityStatement](https://build.fhir.org/ig/hl7au/au-fhir-core/CapabilityStatement-au-core-server.html)
- Change note on Home page to list removed profiles and relate changes - summarise additional other changes.
- Update the following profiles to reference core FHIR resources instead of a removed AU Core profile:
   - AU Core Encounter
   - AU Core MedicationStatement
   - AU Core Procedure
- Remove AU Core Biological Sex Assigned at Birth
- Remove examples for the 9 removed profiles

### PR DESCRIPTION
**Pull request by dbojicic**
Changes:

- Removed profiles:
   -   AU Core BMI
   -   AU Core Diagnostic Imaging Result Observation
   -   AU Core DiagnosticReport
    -   AU Core DocumentReference
    -   AU Core HealthcareService
    -   AU Core Observation
    -   AU Core Oxygen Saturation
    -   AU Core ServiceRequest
    -   AU Core Vitals Panel
- The following profiles now reference core FHIR resources, moving away from previously removed AU Core profiles:
     -  AU Core Encounter
     -  AU Core MedicationStatement
     -  AU Core Procedure

- Additional removal
   -  AU Core Biological Sex Assigned at Birth profile has been removed
    - Examples for the 9 removed profiles have been removed
- Other changes:
   - Removed note on Consent from the Profiles and Extensions page.
   - Updated AU Core Client CapabilityStatement.
   - Updated AU Core Server CapabilityStatement.
   - Updated Comparison with other national and international specifications.
    - Removed terminology resources related to remove profiles from the Terminology page.
     - Updated list of resources that support Provenance from the Provenance profile page.
     - Updated DocumentReference/acp example assertion to removed AU Core DocumentReference example. 

Fixes #106.

### CODE REVIEW

File: input/includes/scope-note.md [hunk 1]
```diff
@@ -1,61 +1,38 @@
 <div class="request-for-feedback" markdown="1">
 
 #### AU Core R1 Scope
```
Comment (dt-r) [inline input/includes/scope-note.md:3]: A request for feedback no longer makes sense here. 
1. Turn it into Note to Contributors that is the difference between last snapshot and this one.
1. Remove all requests for feedback from this page.

File: input/pagecontent/general-requirements.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ There are two different ways to implement AU Core:
 
 Systems may deploy, and support, one or more AU Core profiles (i.e. the profiles governed by this guide) to represent digital health information. A system that implements _profile only support_ will use a profile's content model without any expectations to implement AU Core interactions. 
 
-A system **SHOULD** support all AU Core profiles unless the system does not anticipate supplying or consuming a certain type of data, usually by virtue of playing a limited or specialised role in clinical or information workflows. For example, a pathology laboratory may support [AU Core DiagnosticReport](StructureDefinition-au-core-diagnosticreport.html), but may not support [AU Core MedicationRequest](StructureDefinition-au-core-medicationrequest.html).
+A system **SHOULD** support all AU Core profiles unless the system does not anticipate supplying or consuming a certain type of data, usually by virtue of playing a limited or specialised role in clinical or information workflows.
```
Comment (dt-r) [inline input/pagecontent/general-requirements.md:29]: Replace the example with what is in AU Core Scope. The easy one is to swap AU Core DiagnosticReport for AU Core Pathology Result Observation.

File: input/includes/prov-white-list.md [hunk 3]
```diff
@@ -17,5 +15,4 @@ AU Core Provenance **SHALL** be supported for the following resource types that
 <li>PractitionerRole</li>
 <li>Procedure</li>
 <li>RelatedPerson</li>
```
Comment (dt-r) [inline input/includes/prov-white-list.md:17]: RelatedPerson shouldn't be here.

File: input/ig.xml [hunk 4]
```diff
@@ -1426,79 +1278,6 @@
 			<description value="RelatedPerson: Mr. Peter Rabbit&#xa;&#xa;Relationship to Patient: Emergency Contact and Unrelated Friend of Mr Li Wang"/>
 			<exampleCanonical value="http://hl7.org/fhir/StructureDefinition/RelatedPerson"/>
 		</resource>
-		<!--SERVICEREQUEST EXAMPLES-->
```
Comment (dt-r) [inline input/ig.xml:1429]: There are still RelatedPerson examples - where are these used in other examples?

File: input/includes/prov-white-list.md [hunk 5]
```diff
@@ -4,9 +4,7 @@ AU Core Provenance **SHALL** be supported for the following resource types that
 <li>AllergyIntolerance</li>
 <li>Condition</li>
 <li>DiagnosticReport</li>
```
Comment (dt-r) [inline input/includes/prov-white-list.md:6]: DiagnosticReport should be removed.

**Review by dt-r — CHANGES_REQUESTED** [review]
In addition to specific comments in files the following needs to be addressed:

1. Missing removal of references to Composition, DocumentReference and DiagnosticReport in general-requirements.md
1. Missing removal of the following profile in must-support.md:
    - AU Core Coverage
    - AU Core System Device
    - AU Core HealthcareService	
    - AU Core DiagnosticReport 
    - AU Core RelatedPerson	
    - AU Core ServiceRequest
1. Need to include the CapabilityStatement changes in this PR that are removal of profiles from R1 scope.
1. Remove Lipid Result Observation profile.
1. Remove Coverage search parameter coverage-patient.

File: input/includes/prov-white-list.md (hunk 3 above)
Comment (dbojicic) [reply input/includes/prov-white-list.md]: RelatedPerson is removed now from the list

File: input/includes/scope-note.md (hunk 1 above)
Comment (dbojicic) [reply input/includes/scope-note.md]: Updated note & removed requests for feedback

File: input/ig.xml (hunk 4 above)
Comment (dbojicic) [reply input/ig.xml:1429]: They were referenced by vital signs profiles (height & weight). Removed RelatedPerson examples and updated examples they were referenced by (subject to further change in the example cleanup)

File: input/includes/prov-white-list.md (hunk 5 above)
Comment (dbojicic) [reply input/includes/prov-white-list.md]: Removed now

**Comment by dbojicic** [comment]
Thanks @dt-r, review feedback applied. Please review.

File: input/includes/scope-note.md (hunk 1 above)
Comment (dt-r) [reply input/includes/scope-note.md]: 1. Note should be difference from last snapshot; it isn't a note on the scope changes made by this PR for March for Ballot scope. Change the content and tone to be that diff.
2. confirmed.

File: input/pagecontent/general-requirements.md (hunk 2 above)
Comment (dt-r) [reply input/pagecontent/general-requirements.md]: This comment has not been actioned.

File: input/ig.xml (hunk 4 above)
Comment (dt-r) [reply input/ig.xml:1429]: No need to remove them and change the examples they were still representative of AU Core. Will resolve this PR anyway so we don't spend more time. Majority of examples will change in a forthcoming PR.

**Comment by dt-r** [comment]
> In addition to specific comments in files the following needs to be addressed:
> 
> 1. Missing removal of references to Composition, DocumentReference and DiagnosticReport in general-requirements.md
> 2. Missing removal of the following profile in must-support.md:
>    
>    * AU Core Coverage
>    * AU Core System Device
>    * AU Core HealthcareService
>    * AU Core DiagnosticReport
>    * AU Core RelatedPerson
>    * AU Core ServiceRequest
> 3. Need to include the CapabilityStatement changes in this PR that are removal of profiles from R1 scope.
> 4. Remove Lipid Result Observation profile.
> 5. Remove Coverage search parameter coverage-patient.

1. confirmed fixed
2. Mostly fixed. Still mention of ServiceRequest - replace that guidance text with another resource.
3. TBD after other changes
4. confirmed fixed
5. confirmed fixed
6. remove head circumference profile

File: input/resources/au-core-server.xml [hunk 6]
```diff
@@ -236,42 +236,6 @@
 </tr>
```
Comment (dt-r) [inline input/resources/au-core-server.xml:236]: Fix "http://hl7.org/fhir/R4/composition.html#search" to be "http://hl7.org/fhir/R4/medicationrequest.html#search" - location of this comment is not related to where this fix needs to be applied

File: input/resources/au-core-client.xml [hunk 7]
```diff
@@ -216,42 +216,6 @@
 </tr>
```
Comment (dt-r) [inline input/resources/au-core-client.xml:216]: Fix "http://hl7.org/fhir/R4/composition.html#search" to be "http://hl7.org/fhir/R4/medicationrequest.html#search" - location of this comment is not related to where this fix needs to be applied

**Review by dt-r — CHANGES_REQUESTED** [review]
See comments

File: input/resources/au-core-server.xml (hunk 6 above)
Comment (dbojicic) [reply input/resources/au-core-server.xml:236]: Change applied (not pushed yet).

File: input/resources/au-core-client.xml (hunk 7 above)
Comment (dbojicic) [reply input/resources/au-core-client.xml:216]: Change applied (not pushed yet).

File: input/includes/scope-note.md (hunk 1 above)
Comment (dbojicic) [reply input/includes/scope-note.md]: Change made to be about the difference between the snapshot and this one (not yet pushed).

File: input/pagecontent/general-requirements.md (hunk 2 above)
Comment (dbojicic) [reply input/pagecontent/general-requirements.md]: Have a look at: https://github.com/hl7au/au-fhir-core/pull/110/commits/dc9af83b3bc8a27b6aa08689cdc1249ed1694248#diff-4ce5fd30391d058ae7d545d906e7c4523e7772a36bcf84113e5f328b2141c5dfR29

**Comment by dbojicic** [comment]
> > In addition to specific comments in files the following needs to be addressed:
> > 
> > 1. Missing removal of references to Composition, DocumentReference and DiagnosticReport in general-requirements.md
> > 2. Missing removal of the following profile in must-support.md:
> >    
> >    * AU Core Coverage
> >    * AU Core System Device
> >    * AU Core HealthcareService
> >    * AU Core DiagnosticReport
> >    * AU Core RelatedPerson
> >    * AU Core ServiceRequest
> > 3. Need to include the CapabilityStatement changes in this PR that are removal of profiles from R1 scope.
> > 4. Remove Lipid Result Observation profile.
> > 5. Remove Coverage search parameter coverage-patient.
> 
> 1. confirmed fixed
> 2. Mostly fixed. Still mention of ServiceRequest - replace that guidance text with another resource.
> 3. TBD after other changes
> 4. remove the intro & notes for the lipid profile
> 5. confirmed fixed
> 6. remove head circumference profile

2. Replaced with Immunization.status
3. TBD
4. removed
6. Changes: 
   -  removed profile
   - removed intro & notes md
   - removed from ig.xml
   - removed from comparison.md
   - removed examples
   - marked as removed in the Notes to Contributors

File: input/includes/scope-note.md (hunk 1 above)
Comment (dt-r) [reply input/includes/scope-note.md]: that will do we can fix after merge.

**Comment by dbojicic** [comment]
@dt-r , CapabilityStatement updates are now available for review.
